vos/vos/commands/vcp.py

```diff
@@ -180,17 +183,9 @@ def lglob(pathname):
     def get_md5(filename):
         logging.debug("getting the MD5 for %s" % filename)
         if filename[0:4] == 'vos:':
-            md5 = get_node(filename).props.get('MD5', 'd41d8cd98f00b204e9800998ecf8427e')
+            return get_node(filename).props.get('MD5', 'd41d8cd98f00b204e9800998ecf8427e')
```

Comment:
Why only MD5? I would either have a variable with all the available properties or implement methods (get_md5) in node to hide the key all together.